migrations/20200623141837-upload-library-image-in-gcp.js

```diff
@@ -0,0 +1,61 @@
+module.exports = {
+  async up(db) {
+    
+    const { Storage } = require("@google-cloud/storage");
```

Comment:
@aman-tunerlabs - Make a separate file for image upload to cloud, also it should work with azure or aws or google cloud